docs/src/content/docs/guides/customization.mdx

```diff
@@ -0,0 +1,411 @@
+  ],
+});
+```
+
+  </TabItem>
+</Tabs>
+
+## Social links
+
+Starlight has built-in support for adding links to your social media accounts to the site header via the [`social`](/reference/configuration/#social) option in the Starlight integration.
+
+Currently, links to Discord, GitHub, Mastodon, and Twitter are supported.
+Let us know on GitHub or Discord if you need support for another service!
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+export default defineConfig({
+  integrations: [
+    starlight({
+      title: 'Docs With Social Links',
+      social: {
+        discord: 'https://astro.build/chat',
+        github: 'https://github.com/withastro/starlight',
+        mastodon: 'https://m.webtoo.ls/@astro',
+        twitter: 'https://twitter.com/astrodotbuild',
+      },
+    }),
+  ],
+});
+```
+
+## Edit links
+
+Starlight can display an “Edit page” link in each page’s footer.
+This makes it easy for a reader to find the file to edit to improve your docs.
+For open source projects in particular, this can help encourage contributions from your users.
+
+To enable edit links, set [`editLink`](/reference/configuration/#editlink) in the Starlight integration config.
+The value of `editLink.baseUrl` will be prepended to the path to the current page to form the full edit link.
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+export default defineConfig({
+  integrations: [
+    starlight({
+      title: 'Docs With Edit Links',
+      editLink: {
+        baseUrl: 'https://github.com/withastro/starlight/edit/main/',
+      },
+    }),
+  ],
+});
+```
+
+## Custom CSS styles
+
+Customize the styles applied to your Starlight site by providing additional CSS files to modify or extend Starlight’s default styles.
+
+1. Add a CSS file to your `src/` directory.
+   For example, you could override Starlight’s default blue accent hue to purple:
+
+   ```css
+   /* src/styles/custom.css */
+   :root {
+     --sl-hue-accent: 270;
+   }
+   ```
+
+2. Add the path to your CSS file to Starlight’s `customCss` array in `astro.config.mjs`:
+
+   ```js
+   // astro.config.mjs
+   import { defineConfig } from 'astro/config';
+   import starlight from '@astrojs/starlight';
+
+   export default defineConfig({
+     integrations: [
+       starlight({
+         title: 'Docs With Custom CSS',
+         customCss: [
+           // Path to your custom CSS file, starting with /
+           '/src/styles/custom.css',
+         ],
+       }),
+     ],
+   });
+   ```
+
+## Custom fonts
+
+By default, Starlight uses sans-serif fonts available on a user’s local device for all text.
+This ensures documentation loads quickly in a font that is familiar to each user, without requiring extra bandwidth to download large font files.
+
+If you do need to add a custom font to your Starlight site, this guide will show you how.
```

Comment:
Well if the main aim is just styling Starlight’s output you won’t have any `.astro` files to use scoped style tags in — I think that might fall under the “Astro docs” remit.

Reply:
The “If you do need to…” phrasing was kind of intentionally trying to underline “Don’t do this unless you _need_ to”.

How do you feel about this?

```suggestion
If you do need to add a custom font to your Starlight site, you can set up fonts to use in custom CSS files or with any other [Astro styling technique](https://docs.astro.build/en/guides/styling/).
```

Reply:
I do like it better! I'm not sure that "if you do need to" is as strong a warning in American English as it is in British English, though.  ;) I actually did not get any kind of warning from this statement. So if it's a warning you want...

> Starlight's default fonts have been chosen to work best with documentation. Technical documents often require scannable, nested hierarchies of data in addition to longer-form paragraphs.  If you must add a custom font to your Starlight site, you can set up fonts to use in custom CSS files or with any other [Astro styling technique](https://docs.astro.build/en/guides/styling/).

Reply:
Updated to say “must” instead if “do need to”!

Yes, it is HUGE for a thing we don’t really want to recommend 😓 But I think some people _will_ want it, so I’d rather document it (at the bottom of a page) rather than people thinking it’s not possible. Quite a few of the content on this page could probably be split off into a section of recipe pages probably. We’ll see what the future holds!